traffic_ops/install/bin/postinstall.test.sh

```diff
@@ -0,0 +1,424 @@
+			"config_var": "ldap_conf_location",
+			"hidden": false
+		}
+	],
+	"/opt/traffic_ops/app/conf/ldap.conf": [
+		{
+			"Do you want to set up LDAP?": "no",
+			"config_var": "setupLdap",
+			"hidden": false
+		},
+		{
+			"LDAP server hostname": "",
+			"config_var": "host",
+			"hidden": false
+		},
+		{
+			"LDAP Admin DN": "",
+			"config_var": "admin_dn",
+			"hidden": false
+		},
+		{
+			"LDAP Admin Password": "",
+			"config_var": "admin_pass",
+			"hidden": true
+		},
+		{
+			"LDAP Search Base": "",
+			"config_var": "search_base",
+			"hidden": false
+		},
+		{
+			"LDAP Search Query": "",
+			"config_var": "search_query",
+			"hidden": false
+		},
+		{
+			"LDAP Skip TLS verify": "",
+			"config_var": "insecure",
+			"hidden": false
+		},
+		{
+			"LDAP Timeout Seconds": "",
+			"config_var": "ldap_timeout_secs",
+			"hidden": false
+		}
+	],
+	"/opt/traffic_ops/install/data/json/users.json": [
+		{
+			"Administration username for Traffic Ops": "admin",
+			"config_var": "tmAdminUser",
+			"hidden": false
+		},
+		{
+			"Password for the admin user": "twelve",
+			"config_var": "tmAdminPw",
+			"hidden": true
+		}
+	],
+	"/opt/traffic_ops/install/data/profiles/": [
+		{
+			"Add custom profiles?": "no",
+			"config_var": "custom_profiles",
+			"hidden": false
+		}
+	],
+	"/opt/traffic_ops/install/data/json/openssl_configuration.json": [
+		{
+			"Do you want to generate a certificate?": "yes",
+			"config_var": "genCert",
+			"hidden": false
+		},
+		{
+			"Country Name (2 letter code)": "US",
+			"config_var": "country",
+			"hidden": false
+		},
+		{
+			"State or Province Name (full name)": "Colorado",
+			"config_var": "state",
+			"hidden": false
+		},
+		{
+			"Locality Name (eg, city)": "Denver",
+			"config_var": "locality",
+			"hidden": false
+		},
+		{
+			"Organization Name (eg, company)": "Comcast",
+			"config_var": "company",
+			"hidden": false
+		},
+		{
+			"Organizational Unit Name (eg, section)": "Viper",
+			"config_var": "org_unit",
+			"hidden": false
+		},
+		{
+			"Common Name (eg, your name or your server's hostname)": "cdn",
+			"config_var": "common_name",
+			"hidden": false
+		},
+		{
+			"RSA Passphrase": "testquest",
+			"config_var": "rsaPassword",
+			"hidden": true
+		}
+	],
+	"/opt/traffic_ops/install/data/json/profiles.json": [
+		{
+			"Traffic Ops url": "https://localhost",
+			"config_var": "tm.url",
+			"hidden": false
+		},
+		{
+			"Human-readable CDN Name. (No whitespace, please)": "kabletown_cdn",
+			"config_var": "cdn_name",
+			"hidden": false
+		},
+		{
+			"DNS sub-domain for which your CDN is authoritative": "cdn1.kabletown.net",
+			"config_var": "dns_subdomain",
+			"hidden": false
+		}
+	]
+}
+EOF
+
+"$MY_DIR/postinstall.py" --no-root --root-directory="$ROOT_DIR" --no-restart-to --no-database --ops-user="$(whoami)" --ops-group="$(whoami)" --automatic --cfile="$ROOT_DIR/defaults.json" --debug 2>"$ROOT_DIR/stderr" | tee "$ROOT_DIR/stdout"
```

Comment:
`whoami` for the user and group can be `id -un` and `id -gn`, respectively

Reply:
I'll replace the group one, but the `whoami` manpage says

> Same as id -un.

So there's no reason to replace that one